CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+ - Added free text search
```

Comment:
Hi. Can you add a link to the pr # here? Otherwise this pr looks really good I think.